spdx/v3/v3_0/interfaces_only/model.go

```diff
@@ -0,0 +1,342 @@
+package v3_0
+
+import "time"
+
+type ElementID string
+
+type SpdxDocument interface {
+	ElementCollection
+}
+
+type spdxDocument struct {
```

Comment:
This is the "interfaces only" example, but this is a struct. Is a concrete class a struct and an abstract class an interface?